tvb_build/docker/Dockerfile-migrate

```diff
@@ -0,0 +1,4 @@
+FROM tvb-run
```

Comment:
Do you want to use the **thevirtualbrain/tvb-run** image?

Reply:
Yes

Reply:
Then you have to use full image name thevirtualbrain/tvb-run

Reply:
The image is currently built on Robert's local machine, not on Jenkins. As a next step, we will try to build it on Jenkins and use the existing thevirtualbrain/tvb-run, but then we will need to keep the generated project in a central place.